libs/ui/search/src/lib/results-hits-search-kind/results-hits-search-kind.component.html

```diff
@@ -0,0 +1,25 @@
+<gn-ui-inline-filter
+  [choices]="availableChoices"
+  [selected]="selected"
+  (selectValues)="onSelectedValues($event)"
+>
+  <ng-template let-choice>
+    <gn-ui-kind-badge
+      [kind]="choice.value"
+      [contentTemplate]="customTemplate"
+      class="gn-ui-btn-outline rounded-md py-2 px-2 border-gray-400 cursor-pointer focus-within:text-primary-darkest focus-within:bg-gray-50 transition-colors bg-transparent hover:text-primary-darker focus:text-primary-darker active:text-primary-black"
+      [ngClass]="{
+        '!bg-primary-lightest hover:bg-primary-lightest':
+          selected.includes(choice.value) ||
+          (selected.length === 0 && choice.value === 'all'),
+      }"
+    >
+      <ng-template #customTemplate>
+        {{ 'search.filters.recordKind.' + choice.value | translate
+        }}<span class="ml-1" *ngIf="selected.includes(choice.value)"
+          >({{ choice.count }})</span
+        >
+      </ng-template>
```

Comment:
Thanks for the new UI component @AlitaBernachot ! I was also wondering if the `.includes` logic could move out of the template to TS or be handled by the badge component CSS. But it may finally not make a big difference. I let you judge.

Reply:
Thx for the suggestion, I have moved the logic to the `results-hits-search-kind.ts` TS file.
However I did not moved the logic inside the badge because the badge is dedicated to the display only, the idea is to use this badge everywhere there is a kind badge (search result, record page, ...), for these cases, there is no need for a selected logic.